drivers/interrupt_controller/intc_exti_stm32.c

```diff
@@ -53,6 +53,15 @@ const IRQn_Type exti_irq_table[] = {
 	EXTI9_5_IRQn, EXTI9_5_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn,
 	EXTI15_10_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn
 };
+#elif defined(CONFIG_SOC_STM32F410RX) /* STM32F410RX has no OTG_FS_WKUP_IRQn */
+const IRQn_Type exti_irq_table[] = {
+	EXTI0_IRQn, EXTI1_IRQn, EXTI2_IRQn, EXTI3_IRQn,
+	EXTI4_IRQn, EXTI9_5_IRQn, EXTI9_5_IRQn, EXTI9_5_IRQn,
+	EXTI9_5_IRQn, EXTI9_5_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn,
+	EXTI15_10_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn, EXTI15_10_IRQn,
+	PVD_IRQn, 0xFF, 0xFF, 0xFF,
+	0xFF, TAMP_STAMP_IRQn, RTC_WKUP_IRQn
+};
```

Comment:
This change should be moved to a dedicated driver commit

Reply:
Thanks for rewiewing. How should I manage that? Do a new PR?

Reply:
No need, you can split current commit and force push current branch.

Reply:
But modifying commits and force push as method of choice? Ok I can do...